server/graphql/v2/mutation/MemberInvitationMutations.js

```diff
@@ -58,6 +58,17 @@ const memberInvitationMutations = {
         throw new Forbidden('You can only invite users.');
       }
 
+      const existingMembership = await models.Member.findOne({
+        where: {
+          role: args.role,
+          CollectiveId: account.id,
+          MemberCollectiveId: memberAccount.id,
+        },
+      });
+      if (existingMembership) {
+        throw new BadRequest('Invited user is already a member of the account');
+      }
+
```

Comment:
Could you move this in the `MemberInvitation.invite` method?

Reply:
I believe this check is already present https://github.com/opencollective/opencollective-api/blob/b9aeb1685c08290987a6a67e8e5fe7801d96e261/server/models/MemberInvitation.js#L159-L180

Reply:
Good catch.
There's no issue attached to this, I caught this while demoing last Friday and added to my to-do list.